internal/tiltfile/tiltfile_test.go

```diff
@@ -3900,6 +3900,23 @@ local('echo hi')
 	}
 }
 
+func TestLocalResource(t *testing.T) {
+	f := newFixture(t)
+	defer f.TearDown()
+
+	f.file("Tiltfile", `
+local_resource("test", "echo hi", ["/foo/bar"])
```

Comment:
imo more interesting to pass a relative dependency and make sure it gets converted to an abs path.